src/altinn-app-frontend/webpack.config.docker.js

```diff
@@ -0,0 +1,13 @@
+const common = require('./webpack.config.development');
+
+module.exports = {
+  ...common,
+  devServer: {
+    ...common.devServer,
+    host: '0.0.0.0', // docker needs to bind to public interface in container
+  },
+  watchOptions: {
+    ignored: ['**/node_modules', '**/.git', '**/dist'],
+    poll: 60000,
```

Comment:
Wait, so this polls files every 60s instead of watching files? :thinking: Is this needed because of some side-effect with Windows as the host?

Reply:
Yes, cross platform file watchers was not recommended. I didn’t test.